cfgov/cfgov/urls.py

```diff
@@ -424,9 +424,9 @@
 
     url(r'^_status/', include_if_app_enabled('watchman', 'watchman.urls')),
 
-    flagged_url('FWB_RELEASE',
-                r'^(?i)consumer-tools/financial-well-being/',
-                include('wellbeing.urls')
+    url(
+        r'^(?i)consumer-tools/financial-well-being/',
+        include_if_app_enabled('wellbeing', 'wellbeing.urls')
```

Comment:
I don't think this is necessary. `wellbeing` is inside `cfgov-refresh/cfgov` and it's [not optionally included in settings](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/settings/base.py#L81).

I think this can just be `include('wellbeing.urls')`.

Reply:
Awesome, even simpler